README.md

```diff
@@ -17,8 +17,63 @@ npm install go-ipfs --save
 
 See [IPFS getting-started](http://ipfs.io/docs/getting-started). If anything goes wrong, try using: [http://ipfs.io/docs/install](http://ipfs.io/docs/install).
 
-Note: this module uses the _latest_ version of ipfs. If there is a strong need to vendor an older version, let us know. We care about versions very much, but for a number of reasons, this is easier for us all right now.
-
 ## Development
 
 **Warning**: The binary gets put in the `go-ipfs` folder inside the module folder.
+
+### Which go-ipfs version this package downloads?
+
+Can be specified in `package.json` with a field `go-ipfs.version`, eg:
+
+```json
+"go-ipfs": {
+  "version": "v0.4.5-pre1"
+},
+```
+
+### Using local IPFS daemon as the package download url
+Can be specified in `package.json` with a field `go-ipfs.version`, eg:
+
+```json
+"go-ipfs": {
+  "version": "v0.4.5-pre1",
+  "distUrl": "http://localhost:8080/ipfs/QmSoNtqW22htkg9mtHWNBvZLUEmqfq8su7957meS1iQfeL"
+},
+```
```

Comment:
I'm sure you got it, but just to make clear, this (the distUrl param) is to make it possible to a) test unreleased dist sites (so that whoever publishes one can be sure it doesn't break the whole of JS' go-ipfs ecosystem) and b) can test and use this via IPFS gateway (which makes it *extremely* fast) or locally.